publish2cloud.py

```diff
@@ -176,8 +176,21 @@ def publish_to_s3(config, section, chunknum):
     if config.has_option(section, 's3_key'):
         key = config.get(section, 's3_key')
 
-    chunk_key = os.path.join(
-        config.get(section, os.path.basename('output')), str(chunknum))
+    versioning_needed = (
+        config.has_option(section, 'versioning_needed')
+        and config.getboolean(section, 'versioning_needed')
+    )
+    if versioning_needed and config.has_option(section, 'chunk_key_version'):
```

Comment:
Can we add a test `chunk_key_version` config option to one of the lists in [`sample_shavar_list_creation.ini`](https://github.com/mozilla-services/shavar-list-creation/blob/master/sample_shavar_list_creation.ini)

Reply:
Currently I didn't include any changes on the sample .ini file because the config option is set [here](https://github.com/mozilla-services/shavar-list-creation/pull/99/files#diff-8c5c15d9cbbefc2ec320aa3f2a253290R486) after the master is ran. Adding an empty `chunk_key_version` will create issues on [here](https://github.com/mozilla-services/shavar-list-creation/pull/99/files#diff-e1586d51f03aab5f0d12447ed4102457R184), making the `chunk_key` url return `<list name>//<chunknum>`.
Alternatively, we can add one in and do addition check in the conditional check.